src/components/icon/DsIcon.less

```diff
@@ -1,17 +1,9 @@
+@import (reference) './ds-icon';
+
 :host {
+  &:extend(ds-icon);
```

Comment:
This line plus import of `ds-icon.less`, allows me to define default `<ds-icon>` styles for both Shadow DOM and light DOM (polyfilled browsers elements prior to upgrade).